OsmAnd/src/net/osmand/plus/utils/BLEUtils.kt

```diff
@@ -0,0 +1,31 @@
+package net.osmand.plus.utils
+
+import android.app.Activity
+import android.bluetooth.BluetoothAdapter
+import android.bluetooth.BluetoothManager
+import android.content.Context
+import android.content.pm.PackageManager
+
+object BLEUtils {
+
+	fun isBLEEnabled(activity: Activity?): Boolean {
```

Comment:
Such parameter should not be nullable. 
Why isBLEEnabled returns false if activity null? If activity parameter is null - means that activity is null, but not that BLE is disabled.